packages/sanity/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@next-auth/sanity-adapter",
+  "description": "Sanity Adapter for NextAuth",
+  "version": "1.2.1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "author": "Fedeya <elfedeminaya@gmail.com>",
+  "license": "MIT",
+  "private": false,
+  "keywords": [
+    "next-auth",
+    "oauth",
+    "sanity",
+    "next.js"
+  ],
+  "scripts": {
+    "setup": "yarn build",
+    "build": "tsc"
+  },
+  "files": [
+    "README.md",
+    "dist"
+  ],
+  "devDependencies": {
+    "@types/lru-cache": "^5.1.0"
+  },
+  "dependencies": {
+    "@sanity/client": "^2.8.0",
```

```suggestion
    "@types/lru-cache": "^5.1.0",
    "@sanity/client": "^2.8.0"
  },
  "peerDependencies": {
    "@sanity/client": "^2.8.0"
  },
  "dependencies": {
```

As far as I can tell, `@sanity/client` is only used to enforce types. As such, it could be a dev dependency, but also a peer dependency, as it is required for this adapter to work.